system-tests/services/test-pods.js

```diff
@@ -933,5 +933,166 @@ describe("Services", function() {
+              volumeMounts: [
+                {
+                  name: "test",
+                  mountPath: "test"
+                }
+              ]
+            }
+          ],
+          scaling: {
+            kind: "fixed",
+            instances: 1
+          },
+          networks: [
+            {
+              mode: "host"
+            }
+          ],
+          volumes: [
+            {
+              name: "test"
+            }
+          ]
+        }
+      ]);
+
+      cy.get("button").contains("Review & Run").click();
+
+      cy
+        .root()
+        .configurationSection("General")
+        .configurationMapValue("Service ID")
+        .contains(`/${Cypress.env("TEST_UUID")}/${serviceName}`);
+
+      cy
+        .root()
+        .configurationSection("General")
+        .configurationMapValue("Instances")
+        .contains("1");
+
+      cy
+        .root()
+        .configurationSection("General")
+        .configurationMapValue("CPU")
+        .contains("0.1 (0.1 container-1)");
+
+      cy
+        .root()
+        .configurationSection("General")
+        .configurationMapValue("Memory")
+        .contains("10 MiB (10 MiB container-1)");
+
+      cy
+        .root()
+        .configurationSection("General")
+        .configurationMapValue("Disk")
+        .contains("Not Supported");
+
+      cy
+        .root()
+        .configurationSection("General")
+        .configurationMapValue("GPU")
+        .contains("Not Supported");
+
+      cy
+        .root()
+        .configurationSection("Containers")
+        .configurationMapValue("Container Image")
+        .contains("Not Configured");
+
+      cy
+        .root()
+        .configurationSection("Containers")
+        .configurationMapValue("Force Pull On Launch")
+        .contains("Not Configured");
+
+      cy
+        .root()
+        .configurationSection("Containers")
+        .configurationMapValue("CPUs")
+        .contains("0.1");
+
+      cy
+        .root()
+        .configurationSection("Containers")
+        .configurationMapValue("Memory")
+        .contains("10 MiB");
+
+      cy
+        .root()
+        .configurationSection("Containers")
+        .configurationMapValue("Command")
+        .contains(command);
+
+      cy.root().configurationSection("Storage").then(function($storageSection) {
+        const $tableRow = $storageSection.find("tbody tr:visible");
+        const $tableCells = $tableRow.find("td");
+        const cellValues = ["test", "FALSE", "test", "container-1", "Edit"];
+
+        expect($tableCells.length).to.equal(5);
+
+        $tableCells.each(function(index) {
+          expect(this.textContent.trim()).to.equal(cellValues[index]);
+        });
+      });
+
+      cy.get("button").contains("Run Service").click();
+
+      cy.get(".page-body-content table").contains(serviceName).should("exist");
+
+      cy
+        .get(".page-body-content table")
+        .getTableRowThatContains(serviceName)
+        .should("exist");
```

Comment:
Yup, but I think we decided to use it to be explicit. That was at least where we landed on with one of @wavesoft's PRs